ILRepack.Tests/ILRepack.Tests.csproj

```diff
@@ -55,6 +58,15 @@
     <None Include="packages.config">
       <SubType>Designer</SubType>
     </None>
+    <None Include="sharpdevelop\ICSharpCode.SharpDevelop.BuildWorker120.exe.config" />
```

Comment:
Actually it is, the new test make use of them.
No other/better alternative to committing tons of assemblies/binairies in the repo ?